src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputWhitespaceAround.java

```diff
@@ -0,0 +1,31 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
+
+@SuppressWarnings({"this", "that"})
+public class InputWhitespaceAround
```

Comment:
> Do you mean InputWhitespaceAround should be named InputMethodParamPadWhitespaceAround?

yes.

>  they are in different namespaces

This is another problem we have. Multiple input files are the exact same so users get confused and make changes to the wrong file. This is why we want file names to be named after the test and in it's own directory.